helpers/stringset/string_set.go

```diff
@@ -17,3 +33,21 @@ func (a StringSet) Contains(b StringSet) bool {
 	}
 	return true
 }
+
+func (a StringSet) Difference(b StringSet) StringSet {
```

Comment:
Can you document the behavior, its A - B so it returns anything in B that's not in A